boxes/application/variables.pkr.hcl

```diff
@@ -1,203 +1,209 @@
 variable "cpus" {
-  type = string
+  type    = string
```

Comment:
There seems to be extra space between 'type' and the '=' sign.  Is that intended?

Reply:
Intended. It aligns with the following line. All of the spacing and ordering changes are meant to align with common.pkr.hcl so in it's easier to diff when we vendor node-images again.